docs/configuring-metabase/config-file.md

```diff
@@ -132,6 +134,44 @@ config:
 
 See [Uploads](../databases/uploads.md).
 
+## API keys
+
+You can use the config file to create API keys, which is useful for automated deployments and keeping API keys stable across environments.
+
+You can add API keys like so:
+
+```yaml
+{% raw %}
+version: 1
+config:
+  users:
+    - first_name: Cam
+      last_name: Era
+      password: 2cans3cans4cans
+      email: cam@example.com
+  api-keys:
+    - name: "Admin API key"
+      group: admin
+      description: "API key with admin permissions."
+      creator: cam@example.com
+      key: mb_firsttestapikey
+    - name: "All Users API key"
+      group: all-users
+      description: "API key with permissions of the All Users group."
+      creator: cam@example.com
+      key: mb_secondtestapikey
```

Comment:
I believe we should incentivize people to create keys like we do (with 32 base64 characters), so I'd add examples with keys generated by echo "mb_$(openssl rand -base64 32)"